site/cni-plugin.md

```diff
@@ -30,12 +30,16 @@ To create a network which can span multiple hosts, the Weave peers must be conne
 
 See [Using Weave Net](/site/using-weave.md#peer-connections) for a discussion on peer connections. 
 
-As well as launching Weave Net, you have to run an extra command to
-perform some additional configuration of the Weave bridge:
-
     weave launch <peer hosts>
+
+Then you have to run an extra command to perform some additional
+configuration required by Kubernetes:
+
```

Comment:
But what is the `additional configuration required by Kubernetes`?  That's what's not clear.